dashboard/db/migrate/20190211215513_add_performance_to_teacher_feedback.rb

```diff
@@ -0,0 +1,5 @@
+class AddPerformanceToTeacherFeedback < ActiveRecord::Migration[5.0]
+  def change
+    add_column :teacher_feedbacks, :performance, :string
+  end
+end
```

Comment:
Migrations should always be done as a separate PR, and merged in advance of any code which depends on them. That PR should include the schema.rb changes, as well as any model files that get updated as part of running the migration. To make sure you get the right set of changes, I would recommend:
1. start a new branch off of the staging branch and patch in this migration
2. run `rake db:rollback` (just once) to undo your latest migration
3. run `rake db:migrate`. this should update schema.rb and your model file
4. see comments in this PR on schema.rb and make sure you only include changes related to this PR

The reason for doing migrations separately is because it is very difficult to revert PRs with migrations in them once they start reaching our chef-managed environments (staging, test, levelbuilder, production, etc), so we want to keep them as simple as possible.

Reply:
Created this PR - https://github.com/code-dot-org/code-dot-org/pull/27143